CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 FEATURES:
 * **New Resource:** sumologic_content_permission (GH-340)
+* **New Resource:** sumologic_cse_match_list (GH-353)
```

Comment:
Wouldn't this need to go in the 2.14.0 (unreleased) section?

Reply:
you are completely right,  moved to the right place